src/main/kotlin/com/anaplan/engineering/vdmgradleplugin/Dialect.kt

```diff
@@ -37,35 +37,13 @@ enum class Dialect(val fileExtension : String, val createController : () -> Grad
 }
 
 interface GradleVdm {
-    fun setOutfile(outfile : String)
-
     fun parse(files: List<File>): ExitStatus
 
     fun typeCheck(): ExitStatus
 
     fun getInterpreter(): Interpreter
 }
 
-open class GradleVdmSl : VDMSL(), GradleVdm {
-
-    override fun setOutfile(outfile : String) {
-        VDMJ.outfile = outfile
-    }
-
-}
-
-open class GradleVdmPp : VDMPP(), GradleVdm {
-
-    override fun setOutfile(outfile : String) {
-        VDMJ.outfile = outfile
-    }
-
-}
-
-open class GradleVdmRt : VDMRT(), GradleVdm {
-
-    override fun setOutfile(outfile : String) {
-        VDMJ.outfile = outfile
-    }
-
-}
\ No newline at end of file
+open class GradleVdmSl : VDMSL(), GradleVdm
```

Comment:
Do we even need these any more? I thought they were just here to enable setting the out file programatically.